filter_wander.json

```diff
@@ -0,0 +1,8 @@
+{"newRoute": [
```

Comment:
Maybe rename to something more meaningful.

Also a general note about "Wander", "hiking", etc: Usually it's a good idea to decide on one term and then use that everywhere. It's also usually a good practice to stick to English. So the good choice here IMHO would be to use switzerland-mobility and hiking / cycling everywhere, except in the one place where you need to check `land == 'wander'`.

This may not be very important for such a small project, but in bigger applications like for example the code for map.schweizmobil.ch, these things can cause major headaches.

See also https://martinfowler.com/bliki/TwoHardThings.html